modules/hcp-support-matrix.adoc

```diff
@@ -46,7 +46,7 @@ For hosted clusters, no direct relationship exists between the management cluste
 
 [NOTE]
 ====
-Ensure a maximum latency of 200 ms between the management cluster and hosted clusters.
+Ensure a maximum latency of 200 ms between the management cluster and hosted clusters. This requirement is especially important for multi-architecture deployments, such as when your management cluster is on {aws-short} and your worker nodes are on-premise.
```

Comment:
Nit, [above](https://github.com/openshift/openshift-docs/pull/89809/files#diff-00e6fd9873f49512d35d191b829c178daeab88b1799cfba4b7eb26404985882aR13) you call it a _mixed infrastructure_ when your management cluster is on {aws-short} and your worker nodes are on-premise; here you call it a _multi-architecture_ deployment.

Reply:
Good catch! I'll fix that.